uplink/decorators.py

```diff
@@ -94,9 +94,11 @@ def modify_request(self, request_builder):
 
 # noinspection PyPep8Naming
 class headers(MethodAnnotation):
-    """
-    You can apply this decorator on a class to add headers to all of
-    its API definitions.
+    """Use as a decorator on a class to add Headers to all API calls.
+
+    Attributes:
+        _headers:  A dictionary containing header field as Key
+                   and header values as Value
     """
```

Comment:
Further, when the class has a constructor, let's document the constructor arguments in the class docstring. Checkout [this guide](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), which shows different examples using the Google Python Style Guide. Specifically, take a look at the `ExampleClass`.

Reply:
I've replaced with something like this:
```
    """Use as a decorator on a class to add Headers to all API calls.

    Args:
        arg: A dict containing header input parameters
        kwargs: A dict containing header input parameters
    """
```
Is this what you meant? Thanks for the feedback!

Reply:
Yes! Looks good!